feature/system/gnmi/cliorigin/ate_tests/cli_origin_test/cli_origin_test.go

```diff
@@ -67,26 +67,49 @@ func TestMain(m *testing.M) {
 func TestOriginCliConfig(t *testing.T) {
 	dut := ondatra.DUT(t, "dut")
 	dp1 := dut.Port(t, "port1")
+	var intfAdminDown, intfAdminUp, intfConfig string
 
-	var intfConfig string = `
-interface %s
-  %s
-`
+	switch dut.Vendor() {
```

Comment:
Let's factor the vendor CLIs out to their own functions, in particular so fmt.Sprintf() does not use a variable as a format string which is error prone because the variable can change, whereas the rest of the arguments to fmt.Sprintf() can't change.

```
func juniperCLI(intf string, enabled bool) string {
  op := "disable"
  if enabled {
    op = "enable"
  }
  return fmt.Sprintf(`
interfaces {
  %s {
    %s;
  }
}
`, intf, op)
}

func ciscoCLI(intf string, enabled bool) string { ... } // Similar to above.
func aristaCLI(intf string, enabled bool) string { ... } // Similar to above.
```

Here instead of three string variables, use a single function variable.

```
var cliFn func(intf string, enabled bool) string
switch dut.Vendor() {
case ondatra.JUNIPER:
  cliFn = juniperCLI
case ondatra.CISCO:
  cliFn = ciscoCLI
case ondatra.ARISTA:
  cliFn = aristaCLI
default:
  t.Fatalf("Vendor %v is not supported.", dut.Vendor())
}
```